tests/rbac/server/api/test_signup.py

```diff
@@ -0,0 +1,99 @@
+# Copyright 2018 Contributors to Hyperledger Sawtooth
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# -----------------------------------------------------------------------------
+"""Test the REST API endpoint"""
+# pylint: disable=wildcard-import,unused-wildcard-import,redefined-outer-name,unused-import,invalid-name
+
+import logging
+import json
+import requests
+import pytest
+
+from rbac.common.logs import getLogger
+from tests.rbac.test.fixtures import *
```

Comment:
Minor: But isn't import * discouraged?

Reply:
Yes, absolutely discouraged. This may or may not be good a exception.